Makefile

```diff
@@ -169,6 +177,7 @@ install-mailpit: install-ingress
 
 .PHONY: install-mariadb
 install-mariadb:
+ifeq ($(INSTALL_MARIADB_PROVIDER),true)
```

Comment:
Makefile conditionals are handled by the preprocessor, before parsing. So this logic effectively means that if `INSTALL_MARIADB_PROVIDER` is not `true`, `install-mariadb` is an empty target. This is unusual for a `Makefile` and a bit confusing because you are still going to be executing the target, it's just going to be empty.

I think the more `make` way to do this would be to make the _dependency_ on the `install-mariadb` conditional? Same goes for the other `install-*` targets.

Reply:
Example?

Reply:
Redefine the unconditional `install-lagoon-dependencies` target to only list the unconditional dependencies.

```make
install-lagoon-dependencies: install-registry install-minio install-bulk-storageclass
```

And then conditionally add `install-mariadb` etc.

```make
ifeq ($(INSTALL_MARIADB_PROVIDER),true)
install-lagoon-dependencies: install-mariadb
endif
```

Reply:
NVM